### PR TITLE
Improve errors to show tarball url during failures

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: lib/install.js [hunk 1]
```diff
@@ -241,14 +242,14 @@ function install (gyp, argv, callback) {
 
       req.on('close', function () {
         if (extractCount === 0) {
-          cb(new Error('Connection closed while downloading tarball file'))
+          cb(new Error('Connection closed while downloading tarball file "%s"', tarballUrl))
```
Comment (rvagg) [inline lib/install.js]: I'm pretty sure this isn't going to work, it's not `console.log()`, you'll have to append

Comment (rvagg) [inline lib/install.js]: I guess you mean `VError()` not `Error()`?

**Comment by rvagg** [comment]
I think I'm inclined to agree with @fengmk2 @ https://github.com/rvagg/pangyp/pull/15 that the addition of a new module just for string concatenation might be overkill. verror would be nice if we were chaining errors, but you could use `+` or even `util.format()` if you want to get `console.log()` style formatting. I have no problems with the messages themselves though.

File: lib/install.js (hunk 1 above)
Comment (aredridel) [inline lib/install.js]: Whoops, yes!

**Comment by bnoordhuis** [comment]
Merge conflicts and no activity in over a year so I'll go ahead and close this.

**Pull request closed without merging**